activities/staff-management/onboarding.md

```diff
@@ -30,7 +30,7 @@ The operations coordinator needs to add the new employee to Tentoo NMBRS:
 If the employee agrees to the terms and has no further questions, onboarding takes place as follows:
 
 * Contract is converted from draft to final version and uploaded to Docusign
-* The chief executive, president of the board of directors and the employee need to sign
+* The chief executive, president and the employee need to sign
```

Comment:
The president and chief executive are not roles with signing powers, I recommend:

```suggestion
* Two board of directors members, usually the chairman and secretary, and the employee need to sign
```